fastapi/_compat.py

```diff
@@ -71,8 +71,7 @@
             general_plain_validator_function as with_info_plain_validator_function,  # noqa: F401
         )
 
-    Required = PydanticUndefined
-    Undefined = PydanticUndefined
+    RequiredParam = Undefined = PydanticUndefined
```

Comment:
```suggestion
    RequiredParam = PydanticUndefined
    Undefined = PydanticUndefined
```
This might be a matter of personal preference, but I would use binary assignments instead of tertiary ones. I also think it makes maintenance of code easier on the long run, as you can change one of the assignments and only that particular line will show up in the diff, rather than doing a mix of things on one line.